docs/dev-guide/quickstart.md

```diff
@@ -75,13 +75,16 @@ Run the test suite (Docker must be installed and running):
 
 ```sh
 make test
+# Includes Playwright tests as well (run `make e2e-deps` first)
+make test-all
 ```
 
 Or, to run tests for just a specific language:
 
 ```sh
 make test-js     # Javascript only
 make test-python # Python only
+make test-e2e    # E2E tests only (run `make e2e-deps` first)
 ```
```

Comment:
Both simplifying + expanding a bit:

````suggestion
```sh
make test-all
```

Or, to run just a specific subset of tests:

```sh
make test-js            # Javascript tests (both packages and applications)
make test-prairielearn  # PrairieLearn application tests
make test-python        # Python tests
make test-e2e           # E2E tests (run `make e2e-deps` first)
```
````